docs/contributing.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Contributing
+description: How to contribute to the ProgSoc Docs
+hide:
+    - navigation
+---
+
+# Contributing
+
+## Making Code Changes
+
+There's a few steps to contributing to the ProgSoc docs and a few different ways you can go about it.
```

Comment:
It's probably more straightforward to say, "There are different ways you can contribute to ProgSoc Docs:"